src/test/java/org/pkcs11/jacknji11/CryptokiTest.java

```diff
@@ -152,7 +155,8 @@ public void testGetObjectSizeGetSetAtt() {
         CE.Login(session, CKU.USER, USER_PIN);
         CKA[] templ = {
             new CKA(CKA.CLASS, CKO.DATA),
-            new CKA(CKA.VALUE, "datavalue"),
+            new CKA(CKA.PRIVATE, false),
+            new CKA(CKA.VALUE, "datavalue".getBytes(StandardCharsets.UTF_8)),
```

Comment:
Does it still work if you don't call getBytes with utf-8. Also line 171

 new CKA(CKA.VALUE, "datavalue"),